### PR TITLE
Fix consistency of ipi jobs

### PR DESCRIPTION
**Pull request by derekhiggins**
Remove ipv4 and add ovn-ipv6 to job names as
openshiftsdn and ipv4 are assumed. Also update
the ipi steps to use the chain baremetalds-ipi-test
for consistency.

### CODE REVIEW

**Comment by derekhiggins** [comment]
@stbenjam I meant to get back to this, you mentioned it here
https://github.com/openshift/release/pull/23326#issuecomment-964069309

**Comment by derekhiggins** [comment]
@andfasano @ardaguclu thoughts on this?

**Comment by vrutkovs** [comment]
Needs `core-services/release-controller/_releases/release-ocp-4.*.json` update to mention new job names.
/approve

File: ci-operator/config/openshift/release/openshift-release-master__nightly-4.8.yaml [hunk 1]
```diff
@@ -172,7 +172,7 @@ tests:
         NETWORK_TYPE=OVNKubernetes
         OVN_LOCAL_GATEWAY_MODE=true
     workflow: baremetalds-e2e
-- as: e2e-metal-ipi-compact
+- as: e2e-metal-ipi-compact-ipv6
```
Comment (ardaguclu) [inline ci-operator/config/openshift/release/openshift-release-master__nightly-4.8.yaml:175]: Is it intentional or should be `ovn-ipv6`?

**Review by ardaguclu — COMMENTED** [review]
That looks great to me. I have one comment.

File: ci-operator/config/openshift/release/openshift-release-master__nightly-4.8.yaml (hunk 1 above)
Comment (derekhiggins) [reply ci-operator/config/openshift/release/openshift-release-master__nightly-4.8.yaml:175]: thanks not intentional, will fix

**Comment by derekhiggins** [comment]
> Needs `core-services/release-controller/_releases/release-ocp-4.*.json` update to mention new job names. /approve

thanks, I'll fix it in next update

**Comment by ardaguclu** [comment]
LGTM

I don't add label to @andfasano also check.

**Comment by andfasano** [comment]
Thanks @derekhiggins LGTM but there are some conflicts to be resolved first

**Pull request closed without merging**